e2e-test/src/test/java/io/cdap/plugin/normalize/locators/NormalizeLocators.java

```diff
@@ -0,0 +1,123 @@
+  }
+
+  public static WebElement fieldNormalizingFieldValue(int row) {
+    String xpath = "//*[@data-cy='fieldNormalizing']//*[@data-cy= '" + row + "']" +
+      "//div[@data-cy='multiple-values-input-2']/input";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement outputSchemaDeleteRowButton(int row) {
+    String xpath = "//*[@data-cy='Output Schema']//*[@data-cy='schema-fields-list']" +
+      "//*[@data-cy='schema-row-" + row + "']" +
+      "//button[@data-cy='schema-field-remove-button']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement addRowButton(int row) {
+    String xpath = "//*[@data-cy='schema-row- + " + row + "']//button[@data-cy='schema-field-add-button']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement outputFieldName(int row) {
+    String xpath = "//*[@data-cy='schema-row-" + row + "']//input";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='Output Schema']//*[@data-cy='select-schema-actions-dropdown']")
+  public static WebElement action;
+
+  public static WebElement locatePluginPropertyInvalidInlineError(String propertyName) {
+    return SeleniumDriver.getDriver().findElement(By.xpath("//*[@data-cy='" + propertyName + "']//div[@data-cy=" +
+                                                             "'error-text-0']"));
+  }
+
+  public static WebElement outputSchemaFieldName(int row) {
+    String xpath = "//*[@data-cy='Output Schema']//*[@data-cy='schema-fields-list']//*[@data-cy='schema-row-"
+      + row + "']//input";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static WebElement outputSchemaDataTypeDropdown(int row) {
+    String xpath = "//*[@data-cy='Output Schema']//*[@data-cy='schema-fields-list']" +
+      "//*[@data-cy='schema-row-" + row + "']" +
+      "//*[@data-cy='select-undefined']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  public static By outputSchemaDataTypeOption(int row, String option) {
+    return By.xpath("//*[@data-cy='Output Schema']//*[@data-cy='schema-fields-list']" +
+                      "//*[@data-cy='schema-row-" + row + "']" +
+                      "//*[@data-cy='select-undefined']//*[text()='" + option + "']");
+  }
+
+  public static WebElement outputSchemaAddRowButton(int row) {
+    String xpath = "//*[@data-cy='Output Schema']//*[@data-cy='schema-fields-list']" +
+      "//*[@data-cy='schema-row-" + row + "']" +
+      "//button[@data-cy='schema-field-add-button']";
+    return SeleniumDriver.getDriver().findElement(By.xpath(xpath));
+  }
+
+  @FindBy(how = How.XPATH, using = "(//*[contains(@class, 'metric-records-out-label')])[2]/following-sibling::span")
+  public static WebElement recordsOutCount;
```

Comment:
Remove and use from framework.. this locator available in framework

Reply:
Change Done